newIDE/app/src/EventsFunctionsExtensionEditor/OptionsEditorDialog/ExtensionOptionsEditor.js

```diff
@@ -118,7 +118,7 @@ export const ExtensionOptionsEditor = ({
   return (
     <I18n>
       {({ i18n }: { i18n: I18nType }) => (
-        <ColumnStackLayout noMargin>
+        <ColumnStackLayout noMargin expand>
```

Comment:
for some reason, the fields in that dialog are not full width, but it doesn't seem to have changed recently so maybe we just never noticed?